core/modules/nfv_ctrl.cc

```diff
@@ -206,11 +207,166 @@ CommandResponse NFVCtrl::Init(const bess::pb::NFVCtrlArg &arg) {
     bess::utils::slo_ns = arg.slo_ns();
   }
 
-  return CommandSuccess();
+  if (!arg.port().empty()) {
+    const char *port_name = arg.port().c_str();
+    const auto &it = PortBuilder::all_ports().find(port_name);
+    if (it == PortBuilder::all_ports().end()) {
+      LOG(INFO) << "Failed to find port";
+      return CommandFailure(ENODEV, "Port %s not found", port_name);
+    }
+    port_ = ((PMDPort*)it->second);
+    //create and save the core to bucket mapping
+    for (uint16_t i = 0; i < total_core_count_; i++) {
+      core_bucket_mapping_[i] = std::vector<uint16_t>();
+    }
+    for(uint16_t i = 0; i < port_->reta_size_; i++) {
+      uint16_t core_id = port_->reta_table_[i];
+      assert(core_id < total_core_count_);
+      core_bucket_mapping_[core_id].push_back(i);
+    }
+  }
+  std::ifstream file("long_term_threshold");
+  while (!file.eof()) {
+    uint64_t pps;
+    uint64_t flow_count;
+    file >> flow_count;
+    file >> pps;
+    threshold_[flow_count] = pps;
+  }
+   
+   return CommandSuccess();
+}
+/*
+void WriteToGurobi(uint32_t num_cores, std::vector<float> flow_rates, float latency_bound) {
```

Comment:
You either remove the code or keep it there. We don't keep any unnecessary comment sections.